tests/task.test.ts

```diff
@@ -0,0 +1,23 @@
+import { taskReducer, TaskState, SetCurrentTaskAction, SET_CURRENT_TASK } from '../src/task';
+
+describe('task reducer', () => {
+  test('should return initial state', () => {
+    const expected = {};
+
+    const initialState = undefined;
+    const result = taskReducer(initialState, {} as SetCurrentTaskAction);
+
+    expect(result).toStrictEqual(expected);
+  });
+
+  test('should handle SET_CURRENT_TASK action', () => {
+    const state: TaskState = { currentSid: 'Task1' };
+    const newSid = 'Task2';
+
+    const expected = { ...state, currentSid: newSid };
+
+    const result = taskReducer(state, { type: SET_CURRENT_TASK, newSid } as SetCurrentTaskAction);
```

Comment:
We should use the `setCurrentTaskFromChannel` action creator to generate the action for the test rather than specify it directly

This way we test more of the code and we only need to export the action creator, which is a usual thing to export, rather than the type and type string, which should ideally be kept private to the reducer

Reply:
Looks like this one wasn't implemented?